pkg/plugins/helm/parser.go

```diff
@@ -454,6 +454,10 @@ func (p *Parser) parseCustomValues() (map[string]interface{}, string, string, er
 	if !ok {
 		return nil, "", "", fmt.Errorf("config [Name] is missing")
 	}
+	// Related to https://github.com/helm/helm/pull/1080
+	if len(name) > 14 {
+		return nil, "", "", fmt.Errorf("the length of config [Name] must little than 15")
```

Comment:
less than?

Reply:
Not must, it's a convention. Related to helm/helm#1080.

Reply:
I mean the log content. 
the length of config [Name] must little than 15  -> 
the length of config [Name] must be less than 15